server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -434,19 +435,25 @@ public void testSelectFromTableConstraints() throws Exception {
         execute("SELECT constraint_type, constraint_name, table_name FROM information_schema.table_constraints " +
                 "WHERE table_schema = ?",
             new Object[]{sqlExecutor.getCurrentSchema()});
-        assertEquals(4L, response.rowCount());
+        assertEquals(6L, response.rowCount()); // 4 explicit constraints + 2 NOT NULL derived from composite PK
         assertThat(response.rows()[0][0], is("PRIMARY KEY"));
         assertThat(response.rows()[0][1], is("test_pk"));
         assertThat(response.rows()[0][2], is("test"));
         assertThat(response.rows()[1][0], is("CHECK"));
-        assertThat(response.rows()[1][1], is(sqlExecutor.getCurrentSchema() + "_test_col3_not_null"));
+        assertThat((String) response.rows()[1][1], containsString("not_null"));
```

Comment:
Why use `containsString` and not an exact matching string?

Reply:
> Why use `containsString` and not an exact matching string?

Missed that, rewriting to match - thanks!